inst/examples/parallel.cpp

```diff
@@ -25,9 +25,10 @@ class parallel {
     return ret;
   }
   void update(size_t step, const std::vector<double>& state,
-              dust::RNG<double, int>& rng, std::vector<double>& state_next) {
+              dust::rng_state_t<real_t>& rng_state,
```

Comment:
`step`, `state`, `state_next`, `rng_state` felt like a more natural order to me

Reply:
I always thought that it was "best practice" (whatever that means) to put your writable variables last. We don't *directly* write to rng_state (even though it's modified). I don't feel strongly though - what's your reasoning? Just grouping the state variables?

Reply:
Ah right, I guess I had never come across that rule for C/C++ before. I usually put them in what I consider to be a logical order, so yeah grouping state, grouping rng